src/main/kotlin/io/snyk/plugin/services/SnykCliService.kt

```diff
@@ -116,6 +110,10 @@ class SnykCliService(val project: Project) {
             commands.add(additionalParameters)
         }
 
+        if (additionalParameters == null || !additionalParameters.contains("--file")) {
+            addAllProjectsOption(commands)
```

Comment:
Perhaps you can check if `additionalParameters` contains the `all-projects` flag and add it depending on that instead of only checking if `additionalParameters` is null. I think they should be combinable(?)

Reply:
Yes, we can use additional options with --all-projects. But if additional options contain --file flag we don't use --all-projects. --file flag excludes --all-projects flag.